### ISSUE TITLE
Khimaira Fulmination cant be avoided by terrain elevation atm

### ISSUE THREAD

**Issue by Steam-arch** [EdenServer/community#5069]
### Checklist

- [x] I have searched for existing issues for issues like this one. The issue has not been posted. (Duplicate reports slow down development.)
- [x] I have provided reproducible steps. (No "as the title says" posts please. Provide reproducible steps even if it seems like a waste of time.)
- [ ] I have provided my client version in the details. (type /ver into your game window)
- [x] ***I have provided proof of my claim (era videos, links to era wiki and/or forum posts)*** Lack of proof will slow down fixes and prioritization. Please provide details so that developers' research time is minimized.

### Details

Currently isnt avoidable by terrain elevation. Not sure how fixable elevation issues are.

Era pic, not the common retail kill spot by the small ledge (on the right) but think it shows that terrain elevation will prevent it from hitting.
![image](https://user-images.githubusercontent.com/69089249/169280523-7bee7ba6-898d-401b-b810-18c33859335a.png)
https://www.youtube.com/watch?v=4doLWPL-ENw
![image](https://user-images.githubusercontent.com/69089249/169280908-8b0cf3f6-a3d3-43a6-9a69-18bcdf46e239.png)

To be also mentioned, there might be some funkiness with khimairas resist rates aswell. Like bis rdm/drk can get stun resists after ~5 stuns, while in retail videos rdms can chainspell stun khim without resists, but these are pretty hard to prove. My guess would be that the resist rate just scale slightly too hard or dont decrease over time (?). Slow and para resist rates might also have slight funkiness aswell, but again not sure. Setsu might have something to add to these.

**Comment by ccl085** [comment]
Something was changed during Abyssea (so 2011/out of our era) because of this NM -> https://www.youtube.com/watch?v=vL3ORtro4vA (as you can see on the video people next to the NMs are avoiding AOE).

They made Z axis not matter much so a lot of those thing are prolly not testable on retail, but Zaxis should allow us to avoid most  thing while still being in range.

**Comment by falzum** [comment]
This sounds like 3 different bugs to me. One for khim, one for faf, and one for stun resist rate.

**Comment by Steam-arch** [comment]
@ccl085 Thanks!  Dowzer seemed to confirm your info too. Ledge isnt testable on retail anymore.

@falzum You are right, ill just seperate the faf one for now.

**Comment by ccl085** [comment]
Yes it is everywhere, no idea what they let it for so long than considered it an exploit and fixed it everywhere after level cap increase

**Comment by dowzer** [comment]
Oct. 13, 2010 22:00 [PDT]

As of Oct. 14, 2010, we have confirmed the following issue. Investigation and recovery work is currently underway. We ask for your patience in this matter until the issue has been resolved.

[Issue details]

- In Abyssea - Vunkerl (I-8 ), it is possible to use the terrain to stop the movements of certain Notorious Monsters, enabling the player to take almost no damage in battle.

- Although the use of terrain in battle is not a violation itself, taking advantage of the above issue is considered terrain abuse, which is a violation of the user agreement.

Thank you for your understanding in this matter.



http://www.playonline.com/pcd/verup/ff11us/detail/6035/detail.html

December 2010:

An issue wherein terrain in Abyssea areas could be exploited to battle enemies without taking damage has been resolved.

**Comment by 59blargedy** [comment]
resolved in the last patch
